web3.js/test/keypair.test.ts

```diff
@@ -4,6 +4,12 @@ import {Buffer} from 'buffer';
 import {Keypair} from '../src';
 
 describe('Keypair', () => {
+  it('new keypair', () => {
+    const keypair = new Keypair();
+    expect(keypair.secretKey).to.have.length(64);
```

Comment:
Hmm. I never noticed the JS carries around the expanded secret key. :mortar_board: